doc/cephfs/troubleshooting.rst

```diff
@@ -124,6 +124,9 @@ slow requests are probably the ``mdsc`` and ``osdc`` files.
 * osdc: Dumps the current ops in-flight to OSDs (ie, file data IO)
 * osdmap: Dumps the current OSDMap epoch, pools, and OSDs
 
+If the osdmap or pool are in a NEARFULL condition, then the kernel cephfs
```

Comment:
```suggestion
If the data pool is in a NEARFULL condition, then the kernel
```

Reply:
Looks good. I can't commit it via github web ui, so I'll make the change and re-push.